.github/workflows/update-syscalls.yml

```diff
@@ -0,0 +1,44 @@
+name: Update Syscalls
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '* * * * 2'
+
+jobs:
+  update-syscalls:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        ref: drivers
+        with:
+          fetch-depth: 0
+      
+      - name: Wget syscall-bumper
+        uses: wei/wget@v1.1.1
+        with:
+          args: -O syscalls-bumper_0.0.5_linux_amd64.tar.gz https://github.com/falcosecurity/syscalls-bumper/releases/download/v0.0.5/syscalls-bumper_0.0.5_linux_amd64.tar.gz
```

Comment:
We might want to release `syscalls-bumper` with  a docker image and `:latest` tag, so that we can just use the latest image, without worrying about the versioning!